### DISCUSSION TITLE
Nuxt 3 Website sources

### DISCUSSION THREAD

**Discussion by copostic** [nuxt/nuxt#16235] [Questions]
Hi guys,

There are several elements and libraries integrated on the nuxt.com website that I'd love to use but I cannot find the right repo that contains the sources of the nuxt.com website.

Anyone has any idea about where to find it ?

Thanks !

**Comment by TheAlexLichter** [comment]
`nuxt.com` is powered by the [`docs`](https://github.com/nuxt/framework/tree/main/docs) folder here ☺️
